Bottom: 048094e1ea919fb18e3f871e81e54f52dc171132
Top:    3137b29b865b13ef704b29b1ae0ce3fea586fadd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-25 10:36:40 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/host_vars/vmdaedalus.novatech-llc.com/networking.yml b/ansible-playbook/host_vars/vmdaedalus.novatech-llc.com/networking.yml
new file mode 100644
index 0000000..79bbb96
--- /dev/null
+++ b/ansible-playbook/host_vars/vmdaedalus.novatech-llc.com/networking.yml
@@ -0,0 +1,65 @@
+---
+
+svn_ip_addr: 192.168.0.101/16
+svn_hostname: svn.novatech-llc.com
+svn_port_args:
+    - "{{ svn_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ svn_ip_addr | ipaddr('address') }}:443:443"
+
+gitlab_ip_addr: 192.168.0.102/16
+gitlab_hostname: git.novatech-llc.com
+gitlab_port_args:
+    - "{{ gitlab_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ gitlab_ip_addr | ipaddr('address') }}:443:443"
+    - "{{ gitlab_ip_addr | ipaddr('address') }}:22:22"
+
+wiki_ip_addr: 192.168.0.103/16
+wiki_hostname: wiki.novatech-llc.com
+wiki_port_args:
+    - "{{ wiki_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ wiki_ip_addr | ipaddr('address') }}:443:443"
+
+mantisbt_ip_addr: 192.168.0.104/16
+mantisbt_hostname: mantis.novatech-llc.com
+mantisbt_port_args:
+    - "{{ mantisbt_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ mantisbt_ip_addr | ipaddr('address') }}:443:443"
+
+build_ip_addr: 192.168.0.105/16
+build_hostname: buildsystem.novatech-llc.com
+build_port_args:
+    - "{{ build_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ build_ip_addr | ipaddr('address') }}:443:443"
+
+testssh_ip_addr: 192.168.0.106/16
+testssh_hostname: testssh.novatech-llc.com
+testssh_port_args:
+    - "{{ testssh_ip_addr | ipaddr('address') }}:2200:22"
+
+exim4_ip_addr: 192.168.0.107/16
+exim4_hostname: mail.novatech-llc.com
+exim4_port_args:
+    - "{{ exim4_ip_addr | ipaddr('address') }}:25:25"
+    - "{{ exim4_ip_addr | ipaddr('address') }}:465:465"
+    - "{{ exim4_ip_addr | ipaddr('address') }}:587:587"
+
+openldap_ip_addr: 192.168.0.108/16
+openldap_hostname: ldap.novatech-llc.com
+openldap_port_args:
+    - "{{ openldap_ip_addr | ipaddr('address') }}:389:389"
+    - "{{ openldap_ip_addr | ipaddr('address') }}:636:636"
+
+phpldapadmin_ip_addr: 192.168.0.108/16
+phpldapadmin_hostname: ldap.novatech-llc.com
+phpldapadmin_port_args:
+    - "{{ phpldapadmin_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ phpldapadmin_ip_addr | ipaddr('address') }}:443:443"
+
+phpmyadmin_ip_addr: 192.168.0.109/16
+phpmyadmin_hostname: phpmyadmin.novatech-llc.com
+
+buildbot_ip_addr: 192.168.0.110/16
+buildbot_hostname: buildbot.novatech-llc.com
+buildbot_port_args:
+    - "{{ buildbot_ip_addr | ipaddr('address') }}:80:80"
+    - "{{ buildbot_ip_addr | ipaddr('address') }}:443:443"
